src/components/TransactionOverview/TransactionsOverview.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import transactions from "../../services/api/transactions.json";
+import { StyledOverview } from "./TransactionsOverview.styled";
+
+const getTotal = (type) => {
+  return transactions.filter((el) => el.type === type)
+  .map((el) => el.amount)
+  .reduce((sum, el) => sum + el, 0)
+  .toFixed(2);
+}
+
+export const TransactionOverview = () => {
+  const totalExpenses = getTotal("expense");
+  const totalIncomes = getTotal("income")
+  const currentBalance = (+totalExpenses + +totalIncomes).toFixed(2);
+
+  return (
+    <StyledOverview>
+      <h3>Transactions overview:</h3>
+      <p>Total expenses: $ {totalExpenses}</p>
```

Comment:
you don't have to create separate variable to display totalExpenses 
`<p>Total expenses: $ {getTotalAmount(TRANSACTIONS_TYPES.expense)} </p>`

Reply:
well, I havent noticed it, you can switch back to the previous solution

Reply:
ok